rules-survey/general.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  rules: {
+    // https://eslint.org/docs/rules/semi
+    'semi': ['error', 'always'],
+
+    // https://eslint.org/docs/latest/rules/eqeqeq
+    'eqeqeq': ['error', 'always'],
+
+    // https://eslint.org/docs/latest/rules/comma-dangle
+    'comma-dangle': ['error', 'always-multiline'],
+
+    // https://eslint.org/docs/latest/rules/max-len
+    'max-len': ['error', { 'code': 130, 'ignoreUrls': true }],
+
+    // https://eslint.org/docs/latest/rules/no-console
+    'no-console': 'warn',
```

Comment:
https://eslint.org/docs/latest/rules/no-console

Reply:
It is inconvenient when the application does not compile because of the console.log that you insert to help. Is it possible to simply remove from the build when it goes to production?

Reply:
I believe that it is a much better practice not to use console.log at all but to use Chrome debugger. Then you can immediately see the values that you did not know that you needed when writing the `console.log`